api/specs.proto

```diff
@@ -341,6 +341,11 @@ message NetworkSpec {
 	// swarm internally created only and it was identified by the name
 	// "ingress" and the label "com.docker.swarm.internal": "true".
 	bool ingress = 7;
+
+	// ConfigFrom indicates that the network specific configuration
+	// for this network will be provided via another network, locally
+	// on the node where this network is being plumbed.
+	string config_from = 8;
```

Comment:
@aluzzardi the idea is to create a `config-only` network (Such a network is used only to hold the node-local configuration and not something that can be used by containers directly). These networks are created by the operator for those networks that needs some node-level parameters to tweak.
The user can create a `-config-from=x` network with `--scope=swarm` and that will enable the network to skip the allocator phase of swarmkit and rely on the node-local configuration (x) exposed by the `config-only` network x.